src/chttpd/src/chttpd_db.erl

```diff
@@ -360,10 +357,10 @@ handle_design_info_req(#httpd{method='GET'}=Req, Db, #doc{} = DDoc) ->
 handle_design_info_req(Req, _Db, _DDoc) ->
     send_method_not_allowed(Req, "GET").
 
-create_db_req(#httpd{}=Req, DbName) ->
+create_db_req(#httpd{user_ctx=Ctx}=Req, DbName) ->
     couch_httpd:verify_is_server_admin(Req),
     DocUrl = absolute_uri(Req, "/" ++ couch_util:url_encode(DbName)),
-    case fabric2_db:create(DbName, []) of
+    case fabric2_db:create(DbName, [{user_ctx, Ctx}]) of
```

Comment:
In practice this user_ctx is thrown away here. We check for the server admin then we apply the user_ctx in create then strip it away when we insert it into the fabric2_server cache. In testing or other cases it might be nice to create and get the user_ctx already applied though.

Reply:
@davisp what do you think about this one?